Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -123,7 +123,7 @@
   $SEC101/038.PostgreSqlCredentialsAdoSecret=(?i)(?:password|pwd)\s*=\s*(?P<secret>[^,;"'<\s\n]{8,128})(?:[,;"'<\s\n]|$)
   $SEC101/038.PostgreSqlCredentialsAdoResource=(?i)(?:database|db|dbname)\s*=\s*(?P<resource>[^,;"'<\s\n]+)(?:[,;"'<\s\n]|$)
 
-  $SEC101/041.RabbitMqCredentials=(?i)amqps?:\/\/(?P<id>[^:]+):(?P<secret>[^@]+)@(?P<host>[\w_\-\:]+)\/(?P<resource>[\w]+)(?:[^0-9a-z]|$)
+  $SEC101/041.RabbitMqCredentials=(?i)amqps?:\/\/(?P<id>[^:]+):(?P<secret>[^@\s\n]+)@(?P<host>[\w_\-\:]+)\/(?P<resource>[\w]+)(?:[^0-9a-z]|$)
```

Comment:
>secret [](http://example.com/codeflow?start=67&length=6)

Can we please take a change to the fingerprint class where it throws any exception if anyone introduces newline characters into it? generally, that shouldn't happen, right? this will raise visibility on this class of defect.

remember, spot-fixing a single bug is good. taking a change that allows you to identify every instance of this bug now and moving forward is much better.

Reply:
I rather do this in a separate PR because that can break many other things.
Also, i was just thinking about that, like where should we throw the exception.